src/theory/quantifiers/sygus/cegis_core_connective.cpp

```diff
@@ -407,9 +407,10 @@ void CegisCoreConnective::Component::addRefinementPt(
     Node id, const std::vector<Node>& pt)
 {
   d_numRefPoints++;
-  bool res = d_refinementPt.addTerm(id, pt);
-  // this should always be a new point
-  AlwaysAssert(res);
+  d_refinementPt.addTerm(id, pt);
+  // Note that the above method returns false if pt is a duplicate of
```

Comment:
Which method are you referring to here? Perhaps it is better to move this comment to the documentation of the method in the h file?

Reply:
The comment applies to this context, the method addTerm is already documented (https://github.com/cvc5/cvc5/blob/main/src/expr/node_trie.h#L87).